dist/index.html

```diff
@@ -1,27 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="index.css">
     <title>Capstone</title>
 <script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+
 <body>
     <header class="container">
         <nav>
             <ul>
+                <li id="pokecounter">Pokemon</li>
                 <li>Comments</li>
                 <li>Reservation</li>
-                <li>Likes</li>
             </ul>
         </nav>
     </header>
+
+    <h1>Pokemon, gotta catch 'em all!</h1>
     <div class="itemsContainer"></div>
+
+    <div id="popup">
+        <div class="card">
+            <div id="pokestats">
+
+            </div>
+
+            <div class="section" id="comment-section">
+                <h3 id="comment-counter">
+                    
+                </h3>
+                <ul id="comment-list">
+
```

Comment:
- [ ] As suggested by the previous reviewer please make sure the pokemon count is correct as currently there are 9 cards but show 8. Also before refreshing the page the number shown was 7. One suggestion to fix the problem is using a loader. While the page is loading just show the loader and once its all done you can show the complete page with correct number of cards fetched and everything. This way your site would also look more professional.